featuretools/utils/gen_utils.py

```diff
@@ -1,7 +1,11 @@
 import sys
+from builtins import object
 from collections import Mapping, Set, deque
 from numbers import Number
 
+from past.builtins import basestring
+
+from pympler.asizeof import asizeof as getsize
```

Comment:
The `getsize` method errors on calculating the size on a pandas `SingleBlockManager` object. The [reference in the docstring](https://stackoverflow.com/questions/449560/how-do-i-determine-the-size-of-an-object-in-python) also suggested the `pympler` library so I used that instead